kclvm/src/main.rs

```diff
@@ -23,10 +25,24 @@ fn main() {
             (@arg sort_key: -k --sort "Sort result keys")
             (@arg ARGUMENT: ... -D --argument "Specify the top-level argument")
         )
+        (@subcommand lint =>
```

Comment:
`Lint CLI` may be split into new PRs

Reply:
https://github.com/KusionStack/KCLVM/pull/156